### PR TITLE
hr_worked_days_activity

### PR DESCRIPTION
**Pull request by ghost**
.name_get(context=context) returns a list containning a tuple (id, name). What we want is to display only the name.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: hr_worked_days_activity/hr_activity.py [hunk 1]
```diff
@@ -43,10 +43,14 @@ def _get_activity_name(
 
         for activity in self.browse(cr, uid, ids, context=context):
             if activity.type == 'job' and activity.job_id:
-                res[activity.id] = activity.job_id.name_get(context=context)
+                res[activity.id] = activity.job_id.name_get(
+                    context=context
```
Comment (joaoalf) [inline hr_worked_days_activity/hr_activity.py]: You don't need to pass the context again.

File: hr_worked_days_activity/hr_activity.py [hunk 2]
```diff
@@ -43,10 +43,14 @@ def _get_activity_name(
...
+                )[0][1]
 
             elif activity.type == 'leave' and activity.leave_id:
-                res[activity.id] = activity.leave_id.name_get(context=context)
+                res[activity.id] = activity.leave_id.name_get(
+                    context=context
```
Comment (joaoalf) [inline hr_worked_days_activity/hr_activity.py]: You don't need to pass the context again.

**Comment by joaoalf** [comment]
Hi @dufresnedavid 

Thanks for your contribution.

I've made some comments about your code.

**Comment by ghost** [comment]
@joaoalf Thanks

File: hr_worked_days_activity/hr_activity.py (hunk 2 above)
Comment (ghost) [inline hr_worked_days_activity/hr_activity.py]: @joaoalf En enlevant le context en argument, j'obtien l'erreur suivante dans mon test:
  File "/home/ddufresne/odoo/7.0/git/odoo/addons/hr_holidays/hr_holidays.py", line 99, in name_get
    if not context.get('employee_id',False):
AttributeError: 'NoneType' object has no attribute 'get'
Ça ne cause pas ce problème pour activity.job_id.name_get mais pour être constant, je laisserais le context aux deux endroits.

Comment (yvaucher) [inline hr_worked_days_activity/hr_activity.py]: @dufresnedavid Please add a comment there to explain it. This is an error in core addons.

I created a patch here https://github.com/odoo/odoo/pull/4341

**Comment by ghost** [comment]
Thanks @yvaucher and @joaoalf. I removed the unecessary context in hr_job.name_get() and explained why it is required for hr_holidays_status.name_get().

**Comment by joaoalf** [comment]
@dufresnedavid 
I noticed that in the _get_activity_name() method you don't have a context initialization. Probably if you put a context initialization at the begining of the _get_activity_name() method you may avoid the problem.    

Ex:

<pre>
context = context or {}
</pre>

<pre>
if context is None:
    context = {}
</pre>

**Comment by ghost** [comment]
@joaoalf Of course, I will do it. Thanks

**Comment by yvaucher** [comment]
@dufresnedavid you can remove those contexts https://github.com/odoo/odoo/pull/4341 has been merged

**Comment by max3903** [comment]
@yvaucher merged in version 8, not 7...

**Comment by yvaucher** [comment]
@max3903 uh right, my bad. So I guess it won't be changed any soon in 7.0 I'm doing 8.0 PR anyway. But I won't block this PR for this :+1: 

PR on 7.0: https://github.com/odoo/odoo/pull/4529

**Pull request merged**